pytorch/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM pytorch/pytorch:1.5-cuda10.1-cudnn7-runtime
+
+RUN pip install awscli
+
+RUN pip install tensorboardX==2.0.0 flytekit[pytorch]==0.8.1
```

Comment:
How does TensorBoard work in this case?

Reply:
https://github.com/kubeflow/pytorch-operator/blob/master/examples/mnist/mnist.py#L6

Reply:
hmm we will need to run tensorboard somewhere i guess right?